ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<ol><li>[enhancement] Something I found out recently, apparently you can use greek alphabet in jupyter notebooks using the <code>\</code> command, so <code>\Delta</code> <em>might</em> work (I never tried it in practice).</li><li>re: "decoupled", I'm not sure if it would be too complicated for readers so feel free to say no - do you think that setting this as "partially decoupled (annihilated coulombics, decoupled Lennard-Jones)" might be worth it? (it makes it quite the sentence, so maybe not)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added a sentence describing this.</p>